apps/docs/content/guides/ai/semantic-search.mdx

```diff
@@ -0,0 +1,163 @@
+---
+id: 'ai-semantic-search'
+title: 'Semantic search'
+description: 'Learn how to search by meaning rather than exact keywords.'
+subtitle: 'Learn how to search by meaning rather than exact keywords.'
+sidebar_label: 'Semantic search'
+---
+
+Semantic search interprets the meaning behind user queries rather than exact [keywords](/docs/guides/ai/keyword-search). It uses machine learning to capture the intent and context behind the query, handling language nuances like synonyms, phrasing variations, and word relationships.
+
+## When to use semantic search
+
+Semantic search is useful in applications where the depth of understanding and context is important for delivering relevant results. A good example is in customer support or knowledge base search engines. Users often phrase their problems or questions in various ways, and a traditional keyword-based search might not always retrieve the most helpful documents. With semantic search, the system can understand the meaning behind the queries and match them with relevant solutions or articles, even if the exact wording differs.
+
+For instance, a user searching for "increase text size on display" might miss articles titled "How to adjust font size in settings" in a keyword-based search system. However, a semantic search engine would understand the intent behind the query and correctly match it to relevant articles, regardless of the specific terminology used.
+
+It is also possible to combine semantic search with keyword search to get the best of both worlds. See [Hybrid search](/docs/guides/ai/hybrid-search) for more details.
+
+## How semantic search works
+
+Semantic search uses an intermediate representation called an “embedding vector” to link database records with search queries. A vector, in the context of semantic search, is simply a list of numerical values. They represent various features of the text and allow for the semantic comparison between different pieces of text.
```

Comment:
```suggestion
Semantic search uses an intermediate representation called an “embedding vector” to link database records with search queries. A vector, in the context of semantic search, is a list of numerical values. They represent various features of the text and allow for the semantic comparison between different pieces of text.
```

Reply:
Hmm, this would be another situation where we're trying to demystify the content for newcomers (ie. a vector is not as scary as you think). I'll remove `simply` in this instance, but I suggest we re-think this as a hard rule.